src/wechaty/version.py

```diff
@@ -1,5 +1 @@
-"""
-Do not edit this file.
-This file will be auto-generated before deploy.
-"""
-VERSION = '0.0.0'
+VERSION = '0.6.20'
```

Comment:
This file should not be changed before deployment.

I'll fix the script in Makefile.

Reply:
I'll fix the script in this pr, you can review it later.

Reply:
I have already fix it, please see my last commits.